WI #556: Glossary Terms for 'Correction', 'Credit', and 'Refund' (#673)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/glossary.md

The agreed-upon unit price for a single [Pricing Unit](#pricingunit) of the associated SKU, inclusive of negotiated discounts, if present, and exclusive of any other discounts. This price is denominated in the [Billing Currency](#glossary:billingcurrency).

<a name="glossary:correction"><b>Correction</b></a>

> A correction refers to a charge record applied to a previously invoiced billing period. A correction can be positive (e.g., an additional charge due to late-arriving costs or unbilled usage) or negative (e.g., a [*refund*](#glossary:refund)). Corrections ensure accurate billing by accounting for discrepancies, missed charges, or necessary modifications.

Author: ```suggestion
A charge that serves as a correction to a previously invoiced [*billing period*](#glossary:billing-period), accounting for discrepancies, missed charges, or required changes.
```

The agreed-upon unit price for a single [Pricing Unit](#pricingunit) of the associated SKU, inclusive of negotiated discounts, if present, and exclusive of any other discounts. This price is denominated in the [Billing Currency](#glossary:billingcurrency).

<a name="glossary:correction"><b>Correction</b></a>

> A charge that serves as a correction to a previously invoiced [*billing period*](#glossary:billing-period), accounting for discrepancies, missed charges, or required changes.

Reviewer: ```suggestion
A charge to correct cost or usage data invoiced in a previous [*billing period*](#glossary:billing-period).
```

Reviewer 2: We use "previously invoiced" in a couple of other places (see [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/2d87b7483da258ba5e29278b28f83864b05f886b/specification/columns/chargeclass.md?plain=1#L3) and [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/2d87b7483da258ba5e29278b28f83864b05f886b/supporting_content/columns/chargeclass.md?plain=1#L9)), so I'd recommend we either keep the existing language (my preference) or change it in the other places as well.
```suggestion
A charge to correct cost or usage data in a previously invoiced [*billing period*](#glossary:billing-period).
```

Author: good catch!

<a name="glossary:credit"><b>Credit</b></a>

> A financial incentive or allowance granted by provider independently of other charges to offset costs in various scenarios.

Reviewer: ```suggestion
A financial incentive or allowance granted by a Provider unrelated to other past/current/future charges.
```

Author: nit: 
I suggest we use 'provider' instead of 'Provider' for consistency, as the lowercase version is predominantly used throughout the specification. Currently, 'Provider' appears 64 times (including titles, column display names, column IDs, etc. - likely around 10 occurrences if we exclude those), whereas 'provider' is used 215 times.

Reviewer 2: @ijurica I agree; lower case is fine.  Here's Udam's suggestion with that revision.
```suggestion
A financial incentive or allowance granted by a provider unrelated to other past/current/future charges.
```

An entity that made internal or 3rd party resources and/or services available for purchase.

<a name="glossary:refund"><b>Refund</b></a>

> A refund refers to a reimbursement or credit issued due to billing inaccuracies, service outages, failure to meet Service Level Agreements (SLAs), or any other reason deemed valid under the provider's policies. Refunds may also apply to voluntary cancellations (e.g., unused pre-paid services, reserved capacity, or other up-front commitments) provided the cancellation complies with the provider's refund policy criteria.

Author: ```suggestion
A return of funds that have previously been charged.
```
